eth/scripts/2_setup_contracts.js

```diff
@@ -80,55 +135,44 @@ async function main() {
     azeroContracts.weth,
   );
 
-  // IN-PROGRESS: add pair via a governance Safe action
-  // await most.addPair(wethAddressBytes, wethAddressBytesAzero);
-  const provider = new ethers.JsonRpcProvider(network.config.url);
-
-  const signer0 = signers[1];
-  const safeSdk0 = await createSafeInstance(signer0, contracts);
-
-  console.log("safe owners", await safeSdk0.getOwners());
-  console.log("signer0", signer0.address);
-
-  let iface = await new ethers.Interface([
-    "function addPair(bytes32 from, bytes32 to)",
+  await addTokenPair(wethAddressBytes, wethAddressBytesAzero, [
```

Comment:
this will most likely only be happening on devnet & bridgenet - since for other networks we will not hold the gov keys of the other owners.

Reply:
yes